src/bindings/python/tests/test_onnx/utils/onnx_helpers.py

```diff
@@ -5,8 +5,8 @@
 import onnx
 from onnx.mapping import NP_TYPE_TO_TENSOR_TYPE
 
-from openvino import Core, Tensor
-from openvino.impl import Function
+from openvino.runtime import Core, Tensor
+from openvino.runtime.impl import Function
```

Comment:
> `from openvino.runtime.impl import Function`

`src/bindings/python/src/openvino/runtime/__init__.py` file has this:

```
from openvino.runtime.impl import Function
```

and we don't have `__all__` there.

So, this should work:

```.diff
-from openvino.runtime.impl import Function
+from openvino.runtime import Function
```

or with aliases above

```.diff
-from openvino.runtime.impl import Function
+from openvino import Function
```

with the similar result.

Reply:
> `from openvino.runtime import Core, Tensor, Function`

To make this line work and import all 3 elements we also need to add in `runtime/__init__.py` (besides of added `openvino.Core = Core`):

```
openvino.Tensor = Tensor
openvino.Function = Function
```

Reply:
@alalek  I just wanted to make sure that this way would help with `Core` import. If it helped me, I'd add the rest classes.
BTW, still I need to import firstly `openvino.runtime`, only after that `from openvino import Core` works.
Look at the comment above

Reply:
> 

but it works as it imports like `from openvino.runtime`

Reply:
I agree that we need try to check one part first.

I expected this diff (from initially commented patch):

```.diff
-from openvino.runtime import Core, Tensor
+from openvino import Core
+from openvino.runtime import Tensor
 from openvino.runtime.impl import Function
```

BTW, Could you please add link to builder which contains logs for these lines?

---

> only openvino/__init__.py is initialized 

Looks like you are right (please check separate thread above).